app/api/routes.py

```diff
@@ -245,20 +245,36 @@ def search_results():
 
     # Fetch the filter params from the url, if they were provided.
     paid = request.args.get('paid')
-    filters = ''
+    category = request.args.get('category')
+    language = request.args.get('language')
+    filters = []
 
     # Filter on paid
     if isinstance(paid, str):
         paid = paid.lower()
         # algolia filters boolean attributes with either 0 or 1
         if paid == 'true':
-            filters += 'paid=1'
+            filters.append('paid=1')
         elif paid == 'false':
-            filters += 'paid=0'
+            filters.append('paid=0')
+
+    # Filter on category
+    if isinstance(category, str):
+        filters.append(
+            f"(category:{category} OR category:{category.lower()}"
+            f" OR category:{category.capitalize()})"
+        )
+
+    # Filter on language
+    if isinstance(language, str):
+        filters.append(
+            f"(languages:f{language} OR languages:{language.lower()}"
```

Comment:
I think you have an extra `f` in here before `{language}`. Same as above regarding the `lower()` and `capitalize()` versions

Reply:
Wops my bad